src/components/Auth.tsx

```diff
@@ -0,0 +1,30 @@
+import { signInAnonymously, signOut } from "firebase/auth";
+import { FirebaseError } from "firebase/app";
+import { auth } from "../../firebaseConfig";
+
+// ゲストログインボタンクリック時の処理
+export const guestLogin = async () => {
+  try {
+    await signInAnonymously(auth);
+  } catch (error) {
+    if (error instanceof FirebaseError) {
+      alert(`${error.code}:${error.message}`);
+    } else {
+      alert(`FirebaseErrorで定義されている以外のエラーです：${error}`);
```

Comment:
imo あくまで自分だったらですが、alertってユーザーに見せるものなので、"FirebaseErrorで定義されている以外"とか内部の話を記載しなくていいのかなと思いました！
自分だったら alertとconsole.logを分けて記載しますね。
```suggestion
      alert(`エラーが発生しました：${error.code}`);
      console.log(error.message);
```
問い合わせの想定があるのであれば、問い合わせがあった時にerror.codeでわかるようにしておくのが吉です